src/controllers/monster.js

```diff
@@ -885,6 +888,114 @@ class Monster extends Controller {
+									}
+							const url = this.config.general.scoutURL
+							
+							// Try avoid sending dupes, check current against last used:
+							if (typeof lastcoords === 'undefined' )  {
+								   const lastcoords = coords	// lastcoords not yet defined, set value
+								} else if (lastcoords === coords) {
+								   this.log.info(`[SCOUT] Skipping duplicate attempt with ${coords}`)
+								   break sendscout; 
+								} else {
+								   const lastcoords = coords
+								}
+							
+							try {
+								this.log.info(`[SCOUT] ${data.encounter_id}: ${monster.name} posting to ${url} with ${coords}`)
+								const hrstart = process.hrtime()
+								const timeoutMs = this.config.tuning.dragoniteTimeout || 10000
+								const source = axios.CancelToken.source()
+								const timeout = setTimeout(() => {
+									source.cancel(`[SCOUT] Timeout waiting for scout response - ${timeoutMs}ms`)
+									// Timeout Logic
+								}, timeoutMs)
+
+								const result = await axios.post(url, coords, {cancelToken: source.token}, {headers: headers})
+								clearTimeout(timeout)
+								if (result.status !== 200) {
+									this.log.warn(`[SCOUT] Failed Scout Attempt: Got ${result.status}. Error: ${result.data ? result.data.reason : '?'}.`)
+									return null
+								}
+								const hrend = process.hrtime(hrstart)
+								const hrendms = hrend[1] / 1000000
+
+								if (result.data.includes('<')) { // check for HTML error response
+									this.log.warn(`[SCOUT] Failed Scout Attempt: Got invalid response from Dragonite - ${result.data}`)
+									return null
+								}
+								const scoutResult = result.data
+								this.log.info(`[SCOUT] ${data.encounter_id}: ${monster.name} Scout sent successfully!`)
+								
+							} catch (error) {
+								if (error.response) {
+									this.log.warn(`[SCOUT] Scout Error - Got ${error.response.status}. Error: ${error.response.data ? error.response.data.reason : '?'}.`)
+								} else {
+									this.log.warn(`[SCOUT] Scout NoResponse Error: ${error}.`)
+								}
+								return null
+							}
+							
+							this.log.info(`[SCOUT] Scout clusters: ${this.config.general.scoutClusters}.`)
+							if (this.config.general.scoutClusters) {  //Cluster Scans
+								// Find 6 points around current lat/lon about 70m away from center
+								var degreesPerPoint = 60;
+								var currentAngle = 0;
+									for(var i=0; i < 6; i++)
+									{
+										// X2 point will be cosine of angle * radius (range)
+										var x2 = data.latitude + Math.cos(currentAngle) * ((70*1.732)/1000);
+										// Y2 point will be sin * range
+										var y2 = data.longitude + Math.sin(currentAngle) * ((70*1.732)/1000);
+										const clusterCoord = JSON.stringify([[parseFloat(x2.toFixed(5)),parseFloat(y2.toFixed(5))]])
+											try {
+												this.log.info(`[SCOUT] ${data.encounter_id}: ${monster.name} CLUSTER posting to ${url} with ${clusterCoord}`)
```

Comment:
If you are posting to scout, you can post a list of co-ordinates in one call, and they will all be added to the queue at once.  If you want to hit multiple destinations you should build up an array (with .push) and send all in one go

Reply:
Originally, I had this direction - but got stuck trying to figure out how to push the array